lib/modules/sections.mjs

```diff
@@ -21,6 +21,45 @@ export async function validateAbstractSection (doc, { mode = MODES.NORMAL } = {}
         result.push(new ValidationError('MISSING_ABSTRACT_SECTION', 'The abstract section is missing.', {
           ref: 'https://authors.ietf.org/required-content#abstract'
         }))
+      } else if (!doc.data.content.abstract || doc.data.content.abstract.length < 1) {
+        result.push(new ValidationError('EMPTY_ABSTRACT_SECTION', 'The abstract section is present but contains no meaningful content.', {
+          ref: 'https://authors.ietf.org/required-content#abstract'
+        }))
+      } else {
+        const abstractContent = doc.data.content.abstract.join(' ')
+
+        const rfcReferencePattern = /\[RFC\d+\]/i
+        const urlPattern = /https?:\/\/[^\s]+|www\.[^\s]+/i
+        const sectionReferencePattern = /\bSection\s\d+(\.\d+)?\b|\bAppendix\s\w+\b/i
+        const internetDraftReferencePattern = /\[I-D\.[^\]]+\]/i
+        const numericReferencePattern = /\[\d+\]/i
+        const customReferencePattern = /\[[A-Za-z0-9-]+\]/i
+
+        const validateReference = (pattern, errorCode, errorMessage) => {
+          if (pattern.test(abstractContent)) {
+            switch (mode) {
+              case MODES.NORMAL: {
+                result.push(new ValidationError(errorCode, errorMessage, {
+                  ref: 'https://authors.ietf.org/required-content#abstract'
+                }))
+                break
+              }
+              case MODES.FORGIVE_CHECKLIST: {
+                result.push(new ValidationWarning(errorCode, errorMessage, {
+                  ref: 'https://authors.ietf.org/required-content#abstract'
+                }))
+                break
+              }
+            }
+          }
+        }
+
+        validateReference(rfcReferencePattern, 'INVALID_ABSTRACT_SECTION_REF', 'The abstract section should not contain references to RFCs.')
+        validateReference(urlPattern, 'INVALID_ABSTRACT_SECTION_URL', 'The abstract section should not contain URLs.')
+        validateReference(sectionReferencePattern, 'INVALID_ABSTRACT_SECTION_REF', 'The abstract section should not contain references to sections or appendices.')
+        validateReference(internetDraftReferencePattern, 'INVALID_ABSTRACT_SECTION_ID_REF', 'The abstract section should not contain references to Internet-Drafts.')
+        validateReference(numericReferencePattern, 'INVALID_ABSTRACT_SECTION_NUM_REF', 'The abstract section should not contain numeric references.')
```

Comment:
Please remove the numericReferencePattern and tests  - it is a subset of customReferencePattern and that check is enough.